http-core/src/main/java/org/apache/pekko/http/impl/util/Util.java

```diff
@@ -52,6 +54,26 @@ public static <T, U extends T> scala.Option<U> convertOptionalToScala(Optional<T
         return OptionConverters.toScala((Optional<U>) o);
     }
 
+    // This is needed to be used in Java source code that calls Scala code which expects scala.Long
```

Comment:
Doing this kind of annoyed me. For context this was meant to be solved within `org.apache.pekko.util.OptionConverters` via this PR https://github.com/apache/incubator-pekko/pull/316 however it turns out that `java.lang.Long`/`java.lang.Int` cannot be implicitly converted to `scala.Long`/`scala.Int` when written within Java source code.

While this can be abstracted away for all Scala versions within  `org.apache.pekko.util.OptionConverters` the whole reason why the `java.lang.Long`/`java.lang.Int` methods were added in that PR is because the `scala.jdk.javaapi.OptionConverters` return `java.lang.Long`/`java.lang.Int` and its `scala.jdk.javaapi.OptionConverters` which will be used when Scala 2.12 is going to be dropped.